src/referrals/referrals.service.ts

```
@@ -133,23 +133,29 @@ export class ReferralsService {
     const [rows, total] = await query.getManyAndCount();
 
     const signupCounts = rows.length
-      ? await this.dataSource.query<{ referralEntityId: string; signups: number }[]>(
-          `SELECT ua."referralEntityId", COUNT(DISTINCT ua."userId")::int AS signups
+      ? await this.dataSource.query<{ referralEntityId: string; status: string; count: number }[]>(
```

ESLint

[error] Replace `{·referralEntityId:·string;·status:·string;·count:·number·}[]` with `⏎··········{·referralEntityId:·string;·status:·string;·count:·number·}[]⏎········`

(prettier/prettier)